packages/tsconfig.json

```diff
@@ -14,8 +14,10 @@
 	},
 	"exclude": [
 		"node_modules",
+		"**/stories",
```

Comment:
Nice to find this! I think we can create another follow-up issue for the storybook ts issue.